### DISCUSSION TITLE
Search for subjects, with space in their name

### DISCUSSION THREAD

**Discussion by magirard** [photoprism/photoprism#4111] [Q&A]
I saw another post that mentioned that the "Subject" field is not searchable (Which is a shame because I spent hundreds of hours tagging people), and to just type the name in the search field instead and it'll look into the tags.

The issue I am having with that is that if I type "John Doe", it  does not understand that I need it as a whole and would return results where the subject field could be "Jane Doe" because it seems to search for both 'John' and 'Doe' independently even with the " " around it.

My end goal is to be able to search for a combination of people that were tagged, and that their name appears in the subject field.

so if I can do something like "John Doe" & "Jane Doe" and only get pictures where I can see them both, and not pictures of other Doe family member, that would be great.

**Comment by graciousgrey** [comment]
There is no dedicated search filter for the subject field from the photo details yet. 

However if you tagged people in the people section you can search for them as described here: https://docs.photoprism.app/user-guide/organize/people/#search

**Reply by magirard** [reply]
Can I search for tags as a whole and in combination ?

if a tag contains "John Doe, Jane Doe", is it possible to search for "John Doe" ? I am not talking specifically for the subject field, but in general. It seems searching for "John Doe" search for both word separately.

**Reply by graciousgrey** [reply]
You can find a complete list of available search filters here. It also contains information about which filters support AND OR or wildcard searches.

https://docs.photoprism.app/user-guide/search/filters/#filter-reference
